src/scos_tekrsa/hardware/tekrsa_sigan.py

```diff
@@ -325,11 +327,11 @@ def acquire_time_domain_samples(
                 self.overload = False
                 if "Input overrange" in status:
                     self.overload = True
-                    logger.warning("IQ stream: ADC overrange event occurred.")
+                    logger.info("IQ stream: ADC overrange event occurred.")
```

Comment:
Can we change this to debug. It get's noted in the metadata so it doesn't seem necessary and it can occur multiple times in every acquisition on some sensors.